src/plugins/LinkManager/components/LinkPopover/LinkPopover.tsx

```diff
@@ -0,0 +1,60 @@
+import { Box, Icon } from "@kaizen/component-library"
+import { IconButton } from "@kaizen/button"
+import { Paragraph } from "@kaizen/typography"
+import { Positioner } from "./Positioner.js"
+import { SelectionPosition } from "../../types.js"
+import { usePopover } from "@kaizen/draft-popover"
+import React from "react"
+import editIcon from "@kaizen/component-library/icons/edit.icon.svg"
+import externalLinkIcon from "@kaizen/component-library/icons/external-link.icon.svg"
+import removeLinkIcon from "@kaizen/component-library/icons/remove-link.icon.svg"
+import styles from "./LinkPopover.scss"
+
+export interface LinkPopoverProps {
+  href?: string
+  onRemove: () => void
+  onEdit: () => void
+  selectionPosition: SelectionPosition
+}
+
+export const LinkPopover: React.VFC<LinkPopoverProps> = props => {
+  const { href, onRemove, onEdit, selectionPosition } = props
```

Comment:
Minor - Could line 21 be de-structured in line 20 replacing props with ({ href, onRemove, onEdit, selectionPosition })